test/integration/builder/selects.js

```diff
@@ -1247,5 +1247,141 @@ module.exports = function(knex) {
             });
         });
     });
+
+    it('forUpdate().skipLocked() with order by should return the first non-locked row', function() {
+      // Note: this test doesn't work properly on MySQL - see https://bugs.mysql.com/bug.php?id=67745
+      if (knex.client.driverName !== 'pg') {
+        return;
+      }
+
+      const rowName = 'row for skipLocked() test #1';
+      return knex('test_default_table')
+        .insert([
+          { string: rowName, tinyint: 1 },
+          { string: rowName, tinyint: 2 },
+        ])
+        .then(() => {
+          return knex
+            .transaction((trx) => {
+              // lock only the first row from this test
+              return trx('test_default_table')
+                .where({ string: rowName })
+                .orderBy('tinyint', 'asc')
+                .first()
+                .forUpdate()
+                .then((res) => {
+                  // try to lock the next available row
+                  return knex('test_default_table')
+                    .where({ string: rowName })
+                    .orderBy('tinyint', 'asc')
+                    .forUpdate()
+                    .skipLocked()
+                    .first();
+                });
+            })
+            .then((res) => {
+              expect(res.tinyint).to.equal(2);
+            });
+        });
+    });
+
+    it('forUpdate().skipLocked() should return an empty set when all rows are locked', function() {
+      if (
+        knex.client.driverName !== 'pg' &&
+        knex.client.driverName !== 'mysql'
+      ) {
+        return;
+      }
+
+      const rowName = 'row for skipLocked() test #2';
+      return knex('test_default_table')
+        .insert([
+          { string: rowName, tinyint: 1 },
+          { string: rowName, tinyint: 2 },
+        ])
+        .then(() => {
+          return knex
+            .transaction((trx) => {
+              // lock all of the test rows
+              return trx('test_default_table')
+                .where({ string: rowName })
+                .forUpdate()
+                .then((res) => {
+                  // try to aquire the lock on at least one more row (which isn't available)
+                  return knex('test_default_table')
+                    .where({ string: rowName })
+                    .forUpdate()
+                    .skipLocked()
+                    .limit(1);
+                });
+            })
+            .then((res) => {
+              expect(res).to.be.empty;
+            });
+        });
+    });
+
+    it('forUpdate().noWait() should reject immediately when a row is locked', function() {
+      if (
+        knex.client.driverName !== 'pg' &&
```

Comment:
`!['pg', 'mysql].includes(knex.client.driverName)` might be easier to read, but it's a matter of taste :)

Reply:
It's a bit more symbol-heavy, but it's a trivial change nonetheless. Though every other test already starts like this, with each DB check in a separate line.

Reply:
It's very much a matter-of-taste. Feel free to keep the original version if you like it more or prefer to be consistent.

Reply:
I'll keep it that way, thanks